src/App.css

```diff
@@ -4,9 +4,244 @@
   padding: 0;
   margin: 0;
   box-sizing: border-box;
+  list-style: none;
+  text-decoration: none;
 }
 
 body {
   font-family: 'Poppins', sans-serif;
-  place-content: center;
+  background-color: #f5f6fa;
+  padding: 4%;
+}
+
+header {
+  padding: 24px 4%;
+  background-color: #fff;
+  border-bottom: 1px solid #e8e8e8;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  grid-gap: 40px;
+  gap: 40px;
+}
+
+.logo {
+  color: #0290ff;
+  outline: none;
+}
+
+nav {
+  flex-grow: 1;
+}
+
+.nav-links {
+  display: flex;
+  gap: 40px;
+}
```

Comment:
- [ ] Can you please transform the navbar links to uppercase as given in the project guidelines for getting a replica of the Zeplin's design ? 😃📌

Your design | Given design
---| ---|
![image](https://github.com/HafizMassamTabraizKhan/react-bookstore/assets/106119105/52288d9f-0300-4c72-b5f9-5e5efa192d89) | ![image](https://github.com/HafizMassamTabraizKhan/react-bookstore/assets/106119105/8900d71d-f4e8-4756-af7e-a8f8035e7ad6)